bench/worker/base.rb

```diff
@@ -0,0 +1,56 @@
+require "3scale/backend/job_fetcher"
+
+class Base
+  include Benchmark
+  include TestHelpers::Fixtures
+
+  def initialize
+    super
+    @async = ThreeScale::Backend.configuration.redis.async
+  end
+
+  def async?
+    @async
+  end
+
+  def create_reports(num)
+    warn "generating #{num} transactions.."
+    num.times { default_report }
+  end
+
+  def new_worker(fetcher = nil)
+    Worker.new(async: async?, job_fetcher: fetcher)
+  end
+
+  def clear_queue
```

Comment:
I would also suggest renaming this method to `process_queue` or something. Because now it looks as if it just... cleared it :upside_down_face: 
At first I didn't really get why are we benchmarking the cleaning operation.